reporter/src/main/kotlin/reporters/ExcelReporter.kt

```diff
@@ -92,6 +97,11 @@ class ExcelReporter : Reporter {
         outputDir: File,
         options: Map<String, String>
     ): List<File> {
+        val extraColumns = options[OPTION_EXTRA_COLUMNS].orEmpty()
+            .split(",")
+            .map { it.trim() }
+            .filterNot { it.isEmpty() }
```

Comment:
Nit: Maybe `.filter { it.isNotEmpty() }`?

Reply:
Why?

Reply:
Because at least to me it reads more natural (like natural language), but obviously you disagree as you're asking, so I'm fine with keeping it as-is.

Reply:
No, I agree, done.